desktop/package.js

```diff
@@ -76,12 +77,17 @@ if (version) {
   console.log('Finding electron version')
   exec('npm list --dev electron-prebuilt', (err, stdout, stderr) => {
     DEFAULT_OPTS.version = '0.36.3'
-
     if (!err) {
       try {
-        DEFAULT_OPTS.version = stdout.match(/electron-prebuilt@([0-9.]+)\n/)[1]
-      } catch (ignore) { }
+        DEFAULT_OPTS.version = stdout.match(/electron-prebuilt@([0-9.]+)/)[1]
+        console.log("Found electron-prebuilt version: ", DEFAULT_OPTS.version)
+      } catch (err) {
+        console.log("Couldn't parse npm list to find electron: ", err)
```

Comment:
i think the code i had used was in bash and use plistbuddy to read version from plist but that only works for mac so... I think we should error otherwise we might get bad builds?